src/schema/v2/order/types/PricingBreakdownLines.ts

```diff
@@ -41,10 +41,13 @@ const COPY = {
   },
 } as const
 
-const extractLastOfferAmountFrom = (buyerOrSeller, offers) => {
+const extractLastOfferAmountFrom = (
+  offers: OrderJSON["offers"],
+  from?: "buyer" | "seller"
```

Comment:
Why is this one became optional? We do need to know where offer originates from the be able to interpret it, no?